### PR TITLE
Snapshot

### PR DESCRIPTION
**Pull request by cyrilchukwuebuka**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,20 +3,22 @@
...
 
     <application
+        android:name=".MyApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme"
-        android:name=".MyApplication">
```
Comment (sfxpamusuo) [inline app/src/main/AndroidManifest.xml:12]: Why did you remove this field... android:name

**Review by AmPaschal — APPROVED** [review]
